src/__tests__/TopNav.test.tsx

```diff
@@ -1,65 +1,108 @@
+/**
+ * @jest-environment jsdom
+ */
 import React from "react";
-import { shallow } from "enzyme";
+import { configure, mount, shallow } from "enzyme";
 import TopNav from "../components/TopNav";
+import { IconButton } from "@mui/material";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 
 // TODOs: need to figure out the wrapper type before using the following beforeEach code block
 // let wrapper;
 // beforeEach(()=>{
 // 	wrapper = shallow(<TopNav/>);
 // })
 
+configure({ adapter: new Adapter() });
+
 // test component rendering
 describe("<TopNav /> rendering", () => {
     const wrapper = shallow(<TopNav />);
     // test if the TopNav component renders
     it("should be able to render <TopNav />", () => {
         wrapper;
```

Comment:
Does this code actually test anything? Typically you need to run an expect() + matcher to test code